priv/test_repo/migrations/1_migrate_all.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Absinthe.Ecto.TestRepo.Migrations.MigrateAll do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :username, :string
+    end
+
+    create table(:posts) do
+      add :user_id, references(:users)
```

Comment:
I tried to call this `author_id` instead of `user_id` (and set up the model with `belongs_to :author`, etc.), but for some reason `perform_batch` was failing with:

```
field `Absinthe.Ecto.Post.user_id` in `where` does not exist in the schema in query:

     from p in Absinthe.Ecto.Post,
       where: p.user_id == ^1,
       order_by: [asc: p.user_id],
       select: {p.user_id, p}
```

Not sure if I missed some config, or if there's an issue in this library. Worth looking into.